include/seqan/bam_io/bam_index_bai.h

```diff
@@ -61,6 +61,9 @@
 #ifndef INCLUDE_SEQAN_BAM_IO_BAM_INDEX_BAI_H_
 #define INCLUDE_SEQAN_BAM_IO_BAM_INDEX_BAI_H_
 
+#include <cstdint>
```

Comment:
why this?

Reply:
Oops, I forgot to remove those... I have been trying out different IDEs lately and one gave me all these "Undefined type 'uint32_t' blahblah" errors without these two includes. I will remove them.